docs/core/diagnostics/debug-memory-leak.md

```diff
@@ -26,27 +26,27 @@ In this tutorial, you will:
 
 The tutorial uses:
 
-- .NET Core 3.0 SDK or a later version
+- [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core) or a later version.
 - [dotnet-trace](dotnet-trace.md) to list processes.
 - [dotnet-counters](dotnet-counters.md) to check managed memory usage.
 - [dotnet-dump](dotnet-dump.md) to collect and analyze a dump file.
-- [Sample debug target](sample-debug-target.md) a sample app to diagnose.
+- A [sample debug target](sample-debug-target.md) app to diagnose.
 
 The tutorial assumes the sample and tools are installed and ready to use.
 
 ## Examine managed memory usage
 
-Before you dig into collecting diagnostics data to help us root cause this scenario, you need to convince ourselves that what you are actually seeing is a memory leak (memory growth). You can use the [dotnet-counters](dotnet-counters.md) tool to get at this information.
+Before you start collecting diagnostics data to help us root cause this scenario, you need to make sure that you're actually seeing a memory leak (memory growth). You can use the [dotnet-counters](dotnet-counters.md) tool to confirm that.
 
-Run the [Sample debug target](sample-debug-target.md):
+Open a console window and navigate to the directory where you cloned the samples repository. Run the [sample debug target](sample-debug-target.md):
 
 ```dotnetcli
 # Assumes https://github.com/dotnet/samples was cloned into samples
 cd samples/core/diagnostics/DiagnosticScenarios/
 dotnet run
 ```
```

Comment:
@mairaw, I remember in some PR, you suggested using console in such cases because not all of the commands are .NET CLI.